### DISCUSSION TITLE
Unusable ChatGPT session

### DISCUSSION THREAD

**Discussion by m-farhana** [llm-workflow-engine/llm-workflow-engine#251] [Q&A]
Hi all, 

I keep getting the following error even though I have managed to successfully log into ChatGPT on the FireFox window that pops up after I key in 'chatgpt install'. 

Your ChatGPT session is not usable.
* Run this program with the `install` parameter and log in to ChatGPT.
* If you think you are already logged in, try running the `session` command.

Would really appreciate some help with resolving this. Thank you!

**Comment by thehunmonkgroup** [comment]
Try starting chatgpt with `chatgpt --debug` -- that will show you the browser that chatgpt is using under the hood. Might give you a clue.

**Reply by m-farhana** [reply]
Hi, thank you for replying. I tried this and a FireFox window opens up as shown in the image and I am logged in and everything looks fine so I'm still not sure what the issue is 
<img width="1041" alt="Screenshot 2023-03-24 at 9 15 05 AM" src="https://user-images.githubusercontent.com/86278922/227399708-950ef653-fe7e-4a70-8531-f04e1c4ea22b.png">

**Reply by m-farhana** [reply]
Hello again, I tried again just now and it works. Thanks for your help!
